submission/programs/adlbc.R

```diff
@@ -0,0 +1,228 @@
+###########################################################################
+#' developers : Steven Haesendonckx/
+#' date: 28NOV2022
+#' modification History:
+#' Dadong Zhang, 17DEC2022
+#' Nicole Jones, 12Jan2023
+###########################################################################
+
+# Set up ------------------------------------------------------------------
+rm(list = ls())
+library(haven)
+library(admiral)
+library(dplyr)
+library(labelled) # for variable labelling
+library(sjmisc)
```

Comment:
Instead of manually labelling please make use of the metadata contained in the spec file with {metacore} and {metatools}. See https://github.com/RConsortium/submissions-pilot3-adam/blob/main/submission/programs/adadas.R for an example.